compiler/ztests/const-source.yaml

```
@@ -0,0 +1,36 @@
+script: |
+  export ZED_LAKE=test
+  zed init -q
+  zed create -q test
+  zc -s -C 'const POOL = "test" from POOL' | sed -e "s/[a-zA-Z0-9]\{27\}/XXX/"
+  echo "==="
+  zc -s -C 'const FILE = "A.zson" file FILE'
+  echo "==="
+  zc -s -C 'const URL = "http://brimdata.io" get URL'
+  ! zc -s -C 'const POOL = 3.14 from POOL'
+  ! zc -s -C 'const FILE = 127.0.0.1 file FILE'
+
```

For completeness, maybe add a negative test for `get`?
```suggestion
  ! zc -s -C 'const URL = true get URL'

```